controllers/providers/assistant/gslb.go

```diff
@@ -221,25 +221,50 @@ func (r *GslbLoggerAssistant) InspectTXTThreshold(fqdn string, edgeDNSServerPort
 	return errors.NewResourceExpired(fmt.Sprintf("Can't find split brain TXT record at EdgeDNS server(%s) and record %s ", ns, fqdn))
 }
 
+func GetARecords(msg *dns.Msg) []string {
+	var ARecords []string
+	for _, nsA := range msg.Answer {
+		IP := strings.Split(nsA.String(), "\t")[4]
+		ARecords = append(ARecords, IP)
+	}
+	return ARecords
+}
+
+func DNSQuery(fqdn string, nameserver string, nameserverport int) (*dns.Msg, error) {
+	dnsMsg := new(dns.Msg)
+	edgeDNSServer := fmt.Sprintf("%s:%v", nameserver, nameserverport)
+	fqdn = fmt.Sprintf("%s.", fqdn) // Convert to true FQDN with dot at the end
```

Comment:
should be the argument name fqdn, when it bacomes fqdn here ^^?

Reply:
Not sure if I got the questions right but we need to add dot(.) to the end of the argument to satisfy miekgdns expectations

Reply:
I mean, the argument accepting by DNSQuery is not fqdn. We are making it fqdn.  So the point is, to rename input argument to e.g. `host` ?

Reply:
ah, ok, makes sense

Reply:
@kuritka amended